docs/docs/guides/integrate/identity-providers/introduction.md

```diff
@@ -0,0 +1,182 @@
+
+
+## The advantages of using ZITADEL
+
+The benefits of integrating ZITADEL for managing external IdPs are multiple:
+
+- **No need for custom authentication code**: Your application only needs to interact with ZITADEL, which handles all communications with external IdPs. This abstraction saves you from the complexity of direct integration with multiple IdPs.
+
+
+- **Unified protocol handling**: ZITADEL abstracts away the specific protocols used by different IdPs. Your application communicates with ZITADEL using a standard protocol (e.g., OpenID Connect), while ZITADEL takes care of the rest.
+
+
+- **Centralized user management**: All user profiles are managed within ZITADEL, allowing for a unified view of user identities regardless of the IdP used for authentication.
+
+
+- **Dynamic profile synchronization**: When users update their profiles on an external IdP, those changes are reflected in ZITADEL at the next login, ensuring that user data remains current.
+
+
+- **Simplified account linking**: ZITADEL can link identities from multiple IdPs to a single user profile, facilitating a cohesive user experience across different authentication methods.
+
+
+- **Enhanced group and policy management**: With support for group synchronization and detailed policy configurations, ZITADEL offers granular control over user access and authentication processes.
+
+
+
+## The user journey 
+
+- **Integration with external identity providers**: ZITADEL supports integrating a variety of external identity providers, including social logins like Google or GitHub, as well as custom IdPs that use OpenID Connect or SAML protocols.
+
+
+- **Initiation of sign-in process**: Users select to sign in with an external IdP within your application.
+
+
+- **Redirection to ZITADEL**: The application redirects the user to ZITADEL, which guides them to their chosen external IdP for authentication.
+
+
+- **User authentication**: After successfully logging in at the external IdP site, users are redirected back to the application through ZITADEL, bringing along authentication tokens and profile information.
+
+
+- **Account linking for new users**: If the user's identity from the external IdP does not exist in ZITADEL, they're presented with two options before moving forward:
+  - **Create a new account**: Choosing this option creates a new ZITADEL account linked to the external IdP.
+  - **Autolinking with an existing local account**: Users have the option to link their new external identity to an existing local account in ZITADEL, enabling future logins with either their local account or the external IdP.
```

Comment:
We do not provide autolinking, they need to link manually by authenticating the user to whom they want to link

Reply:
Done.